docs/html/development/ci.rst

```diff
@@ -69,7 +69,7 @@ provides free executors for open source packages:
 
 .. _`Travis CI`: https://travis-ci.org/
 .. _`Appveyor CI`: https://www.appveyor.com/
-.. _`Azure DevOps CI`: https://dev.azure.com/
+.. _`Azure DevOps CI`: https://azure.microsoft.com/en-us/services/devops/
```

Comment:
I believe the locale part (`en-us`) can be dropped and microsoft.com would redirect based on browser locale.